shell/platform/embedder/embedder.h

```diff
@@ -944,85 +957,195 @@ FlutterEngineResult FlutterEngineSendPlatformMessage(
+///
+/// @param[in]  engine         A running engine instance.
+/// @param[in]  data_callback  The callback invoked by the engine when the
+///                            Flutter application send a response on the
+///                            handle.
+/// @param[in]  user_data      The user data associated with the data callback.
+/// @param[out] response_out   The response handle created when this call is
+///                            successful.
+///
+/// @return     The result of the call.
+///
 FLUTTER_EXPORT
 FlutterEngineResult FlutterPlatformMessageCreateResponseHandle(
     FLUTTER_API_SYMBOL(FlutterEngine) engine,
     FlutterDataCallback data_callback,
     void* user_data,
     FlutterPlatformMessageResponseHandle** response_out);
 
-// Collects the handle created using
-// |FlutterPlatformMessageCreateResponseHandle|.
+//------------------------------------------------------------------------------
+/// @brief      Collects the handle created using
+///             `FlutterPlatformMessageCreateResponseHandle`.
+///
+/// @see        FlutterPlatformMessageCreateResponseHandle()
+///
+/// @param[in]  engine     A running engine instance.
+/// @param[in]  response   The platform message response handle to collect.
+///                        These handles are created using
+///                        `FlutterPlatformMessageCreateResponseHandle()`.
+///
+/// @return     The result of the call.
+///
 FLUTTER_EXPORT
 FlutterEngineResult FlutterPlatformMessageReleaseResponseHandle(
     FLUTTER_API_SYMBOL(FlutterEngine) engine,
     FlutterPlatformMessageResponseHandle* response);
 
+//------------------------------------------------------------------------------
+/// @brief      Send a response from the native side to a platform message from
+///             the Dart Flutter application.
+///
+/// @param[in]  engine       The running engine instance.
+/// @param[in]  handle       The platform message response handle.
+/// @param[in]  data         The data to associate with the platform message
+///                          response.
+/// @param[in]  data_length  The length of the platform message response data.
+///
+/// @return     The result of the call.
+///
 FLUTTER_EXPORT
 FlutterEngineResult FlutterEngineSendPlatformMessageResponse(
     FLUTTER_API_SYMBOL(FlutterEngine) engine,
     const FlutterPlatformMessageResponseHandle* handle,
     const uint8_t* data,
     size_t data_length);
 
-// This API is only meant to be used by platforms that need to flush tasks on a
-// message loop not controlled by the Flutter engine. This API will be
-// deprecated soon.
+//------------------------------------------------------------------------------
+/// @brief      This API is only meant to be used by platforms that need to
+///             flush tasks on a message loop not controlled by the Flutter
+///             engine.
+///
+/// @attention  This API will be deprecated and is not part of the stable API.
```

Comment:
You use `@deprecated` elsewhere; why not here?

Reply:
Fixed.